modules/engine/package.json

```diff
@@ -35,14 +35,15 @@
   ],
   "sideEffects": false,
   "scripts": {
-    "build-bundle": "ocular-bundle ./src/index.ts",
-    "pre-build": "npm run build-bundle && npm run build-bundle -- --env=dev"
+    "build-minified-bundle": "ocular-bundle ./src/index.ts -output=dist/dist.min.js",
+    "build-dev-bundle": "ocular-bundle ./src/index.ts -output=dist/dist.dev.js ----env=dev",
+    "prepublishOnly": "npm run build-minified-bundle && npm run build-dev-=bundle"
   },
-  "dependencies": {
-    "@babel/runtime": "^7.0.0",
-    "@luma.gl/constants": "9.0.0-beta.4",
+  "peerDependencies": {
     "@luma.gl/core": "9.0.0-beta.4",
```

Comment:
Do not lock the peer dependency versions. I.e. `^9.0.0`. These are not managed by lerna during publish.